samples/AnomalyDetector/Generated/Docs/AnomalyDetectorClient.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <doc>
   <members>
-    <member name="DetectUnivariateEntireSeriesAsync(RequestContent,RequestContext)">
+    <member name="DetectUnivariateEntireSeriesAsync(global::Azure.Core.RequestContent,global::Azure.RequestContext)">
```

Comment:
we want this to go in the summary on the method itself not in the xml doc file.

Reply:
Or maybe we could have both?

Reply:
MSDoc will retrieve <summary> in client file, and the external xml doc. We may need add this one place, othewise, msdoc may have duplicate instruction.

@m-nash I have create a PR (https://github.com/Azure/autorest.csharp/pull/3334) to add summary in the method itself, and follow the format described in the email. Would you please help to review that.